test/Debugger/failfast.js.dbg.baseline

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
nit: rename this file from failfast.js.dbg.baseline to empty.dbg.baseline so that it can be reused as empty file in future.